docs/sphinx/quickstart.rst

```diff
@@ -179,14 +179,19 @@ build an executable using |cmake| and |hpx|:
    project(my_hpx_project CXX)
    find_package(HPX REQUIRED)
    add_executable(my_hpx_program main.cpp)
-   target_link_libraries(my_hpx_program HPX::hpx HPX::iostreams_component)
+   target_link_libraries(my_hpx_program HPX::hpx HPX::wrap_main HPX::iostreams_component)
 
 .. note::
 
    You will most likely have more than one ``main.cpp`` file in your project.
    See the section on :ref:`using_hpx_cmake` for more details on how to use
    ``add_hpx_executable``.
 
+.. note::
+
+   ``HPX::wrap_main`` is required if you are implicitly using ``main`` as the
```

Comment:
This is 'required' on Linux/MacOS only. But I guess is doesn't do any arm on other systems

Reply:
Yeah, I know, but I'd rather that users link to it anyway since that'll make things work everywhere. On top of that it would give us the freedom to move `hpx/hpx_main.hpp`/`hpx/wrap_main.hpp` to a separate include directory so that not linking to `HPX::wrap_main` would already be a compiler error rather than a linker error.